_posts/2015-06-12-topics.md

```diff
@@ -871,3 +878,238 @@ This topic is deprecated. Please use [shipment:packed](#shipment_packed) instead
     }
 }
 ```
+<h3 id="tracker_updated">
+    tracker:updated
+</h3>
+
+```json
+{
+    "topic" : "tracker:updated",
+    "message" : {
```

Comment:
If an update occurred, what data was updated? Add:
- old_status
- new_status
- new_details (array of detail objects added in this update)
- delivered_at (may be null if not yet delivered)
- signed_by (may be null)